### ISSUE TITLE
Lint docs for specific words

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-docs#1904]
> 💡 We could use https://vale.sh/ to lint our docs for such words and instead of failing the CI we could use it to warn the author and reviewers of the usage with a suggestion about when to use which form.

_Originally posted by @korthout in https://github.com/camunda/camunda-platform-docs/pull/1900#discussion_r1158669767_

**Comment by akeller** [comment]
Something to think about @christinaausley and @pepopowitz! Could be a quick win to implement and hugely valuable to reduce some of the manual work Christina does for reviews, specifically reviews that include certain terms (upgrade vs. update or casing on 3rd party terms).

**Comment by korthout** [comment]
Another case where this could be applied is between American and British English styles. No need to do it all at once but I'm just highlighting the potential.

**Comment by pepopowitz** [comment]
Yes, this would be great! 

I was originally thinking I'd add this as an item to https://github.com/camunda/camunda-platform-docs/issues/1742, but I stopped, because that one tracks reducing friction for _contributors_, not maintainers. I will peruse the list of open issues soon to see if there are enough similar issues to this, in which case I'll create another epic to track reducing friction for maintainers.

**Comment by pepopowitz** [comment]
> Yes, this would be great!
> 
> I was originally thinking I'd add this as an item to #1742, but I stopped, because that one tracks reducing friction for _contributors_, not maintainers. I will peruse the list of open issues soon to see if there are enough similar issues to this, in which case I'll create another epic to track reducing friction for maintainers.

Nope, I lied. I _am_ adding this to #1742, because automating this would also tighten the contributor's feedback loop -- they can go back and fix these things before @christinaausley even gets to her review.

**Comment by pepopowitz** [comment]
Note that vale has been added to this project in a `check-format` workflow, as part of #2675. That makes the work for this issue a little easier, as we need only configure to the rules we want.

**Comment by akeller** [comment]
@christinaausley the foundation is ready for any additional Vale work. This would be an excellent tool for you to have in your tech writer toolbox and make your reviews easier (plus strengthen your partnership with the robot overlords). And so timely! https://camunda.slack.com/archives/C026U8GBNSW/p1707935866675739

**Comment by christinaausley** [comment]
@pepopowitz wanna pair on this after our authentication/tutorial work? Assigning this to myself for now, but if you're interested feel free to assign yourself as well.
